### DISCUSSION TITLE
New icon proposal

### DISCUSSION THREAD

**Discussion by setanarut** [sbarex/SourceCodeSyntaxHighlight#121] [Show and tell]
I made a better icon, you can update it if you want.

![SH-icon](https://user-images.githubusercontent.com/36481442/141824237-93636934-af01-4600-a387-6d1a7606c5c9.png)

sketch file
[icon.sketch.zip](https://github.com/sbarex/SourceCodeSyntaxHighlight/files/7540369/icon.sketch.zip)

**Comment by sbarex** [comment]
Thanks a lot! 

What do you think if the quicklook eye icon is also added? 

Like that: 
![shicon1](https://user-images.githubusercontent.com/1948940/141939966-ade1953f-fb13-49cb-84aa-d529a20417f5.png)

**Reply by setanarut** [reply]
It is nice. A smaller version is also required.
<img width="241" alt="LWScreenShot 2021-11-17 at 00 39 34" src="https://user-images.githubusercontent.com/36481442/142070101-56f742b3-7a41-446c-b524-fc047f1f3fd9.png">

**Comment by setanarut** [comment]
the yellow line is very close to the eye.

**Comment by sbarex** [comment]
new test: 

![icon_512x512@2x@2x](https://user-images.githubusercontent.com/1948940/142773550-77df31a7-c536-48d1-bf2c-942e58fa0f7e.png)

**Reply by setanarut** [reply]
Nice, but i don't like bold gray border. I think borderless is better.
